rhino/src/main/java/org/mozilla/javascript/IRFactory.java

```diff
@@ -854,7 +851,6 @@ private Node transformObjectLiteral(ObjectLiteral node) {
             }
         }
         object.putProp(Node.OBJECT_IDS_PROP, properties);
```

Comment:
The `properties` are a mixture of `String`, `Integer` and `Node`, where `Node` is Computed Property Name.